package.json

```diff
@@ -47,16 +47,17 @@
 		"@prismicio/mock": "0.0.10",
```

Comment:
This doesn't have to happen in this PR, but is it possible to update `@prismicio/mock` to the latest version and unpin it?

Reply:
👍